CONTRIBUTING.md

```diff
@@ -0,0 +1,172 @@
+# Contributing Guide
+
+* [New Contributor Guide](#contributing-guide)
+  * [Ways to Contribute](#ways-to-contribute)
+  * [Find an Issue](#find-an-issue)
+  * [Ask for Help](#ask-for-help)
+  * [Pull Request Lifecycle](#pull-request-lifecycle)
+  * [Development Environment Setup](#development-environment-setup)
+    * [Requirements](#requirements)
+    * [Build](#build)
+    * [Install](#install)
+    * [Test](#test)
+  * [Pull Request Checklist](#pull-request-checklist)
+
+Welcome! We are glad that you want to contribute to our project!
+
+As you get started, you are in the best position to give us feedback on areas of
+our project that we need help with including:
+
+* Problems found during a new developer environment setup.
+* Gaps in our Quickstart Guide or documentation.
+* Bugs in our automation scripts.
+
+If anything doesn't make sense, or doesn't work when you run it, please open a
+[issue](https://github.com/rpm-software-management/dnf5/issues/new) and let us know!
+If you file a bug on Fedora you could open a bug on
+[Bugzilla](https://bugzilla.redhat.com/enter_bug.cgi?product=Fedora&component=dnf5).
+
+## Ways to Contribute
+
+We welcome many different types of contributions including:
+
+* [New features](https://github.com/rpm-software-management/dnf5/labels/RFE)
+* [Bug fixes](https://bugzilla.redhat.com/buglist.cgi?bug_status=__open__&product=Fedora&component=dnf5)
+* [Documentation](https://dnf5.readthedocs.io)
+* [Issue Triage](https://github.com/rpm-software-management/dnf5/issues)
+* [Copr Builds](https://copr.fedorainfracloud.org/coprs/rpmsoftwaremanagement/), [CI/CD](https://github.com/rpm-software-management/dnf5/actions)
+* [Communications](#ask-for-help) / [GH Discussions](https://github.com/rpm-software-management/dnf5/discussions)
+
+## Find an Issue
+
+We have good first issues for new contributors and help wanted issues suitable
+for any contributor. Issues labeled as
+[good first issue](https://github.com/rpm-software-management/dnf5/labels/good%20first%20issue)
+are issues suitable for someone who isn't a core maintainer and wants to get familiar with the codebase.
```

Comment:
```suggestion
If you are a new contributor, you can search for issues with the [_good first issue_](https://github.com/rpm-software-management/dnf5/labels/good%20first%20issue) label. These issues are suitable for contributors who are not core maintainers and want to get familiar with the codebase.

If you are a regular contributor, you can search for issues with the _help wanted_ label.
```

Here, I'm not sure if I interpreted the info in the correct way :-D Also, would it be possible to add also the link to the _help wanted_ issues search result similarly to what you have for _good first issue_?

Reply:
you are correct; however, we have a change in the workflow now, so I will simply remove the last line "If you are a regular contributor... wanted label." since this piece of information is not valid anymore

Reply:
Oki, got it :-)